classes/Feature.php

```diff
@@ -62,34 +66,38 @@ class FeatureCore extends ObjectModel
      *
      * @param int $idLang    Language ID
      * @param int $idFeature Feature ID
+     * @param boolean $display Featue display
      *
      * @return array Array with feature's data
      */
-    public static function getFeature($idLang, $idFeature)
+    public static function getFeature($idLang, $idFeature, $display = true)
```

Comment:
I get it, we can now filter by the value of column `display`.
I have not tested your code yet, but do we have a way to get all Features, whatever their visibility ? (And is there this need ?)